### ISSUE TITLE
[IMPORTANT] Ventus/Nunera DMCA Takedown Request.

### ISSUE THREAD

**Issue by angelicwine** [NuneraRebirth/Nunera.github.io#1]
### Check these boxes to make sure you did everything right!

- [X] I have made sure that this issue does not already exist
- [X] I have submitted screenshots of the error
- [X] You are not using the website on any clones (you are using 3kh0.github.io and nothing else)
- [X] I just checked all of these boxes without reading the labels

### What browser are you using?

Google Chrome

### What device are you using?

Other (please note in text field below)

### Please explain the issue

Hello, I am coming to notify that your account will be DMCA'ed due to not following our last warning. In 48 hours from 5:10 CST, a DMCA takedown will begin. In order to stop this, delete all forks of 3kh0 or join the discord. "https://discord.gg/3kh0"

**Comment by angelicwine** [comment]
.

**Comment by ghost** [comment]
Right, thank you Kaio. Join the Discord and let us know your username so we can add you into the ticket.

**Comment by angelicwine** [comment]
Hello, You should be grateful we gave you more time, the DMCA process will start tomorrow.
